README.md

```diff
@@ -208,6 +208,15 @@ View additional information on configuring Windows images for bootstrap in the d
 Users can also include the ec2 server id in the node name by placing `%s` in the string passed to the `--chef-node-name` option. The %s is replaced by the ec2 server id dynamically. 
 e.g. `-N "www-server-%s" or  --chef-node-name "www-server-%s"`
 
+#### Tagging node in Chef
+
+Users can use option `--tag-node-in-chef` for tagging node in EC2 and chef both with `knife ec2 server create` command. If user does not pass option then it will be tagged only in EC2.
```

Comment:
Small change: `If user does not pass this option, then the node will be tagged only in EC2.`